.dockerignore

```diff
@@ -1,6 +1,8 @@
 .gitignore
 Dockerfile
 env
+venv
 *.db
 data
-panopticum_django/settings_local.py
\ No newline at end of file
+panopticum_django/settings_local.py
+panopticum/migrations
```

Comment:
We doesn't need migrations at clean install. Anyway IMHO migrations it's environment factor. For different instance need different migrations set.